### ISSUE TITLE
Custom builds

### ISSUE THREAD

**Issue by paulmillr** [paulmillr/es6-shim#136]
Would be really nice to have lodash-like custom builds, I guess. Some people need only 1-3 functions from here.

**Comment by chrishyle** [comment]
:+1:

**Comment by monolithed** [comment]
Are there any plans?

**Comment by ljharb** [comment]
I think this would need to happen after #297.
